dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/beans/factory/annotation/ReferenceAnnotationBeanPostProcessor.java

```diff
@@ -250,29 +354,12 @@ private boolean existsServiceBean(String referencedBeanName) {
 
     }
 
-    private boolean isRemoteReferenceBean(ReferenceBean referenceBean, AnnotationAttributes attributes) {
-        boolean remote = Boolean.FALSE.equals(referenceBean.isInjvm()) || Boolean.FALSE.equals(attributes.get("injvm"));
+    private boolean isRemoteReferenceBean(AnnotationAttributes attributes) {
+        //TODO injvm 为空值时是否可以走本地调用？
```

Comment:
pls comment in english

It would be more reasonable to make it configurable. 
There is a related issue #6842